File: src/com/riadsafowan/java/Practice2.java
package com.riadsafowan.java;

import java.util.Scanner;

public class Practice2 {

    public static void main(String[] args) {
//        1.
        int num;
        Scanner scanner = new Scanner(System.in);
        System.out.print("Enter a number: ");
        num = scanner.nextInt();
        System.out.println("Is the number " + num + " divisible by 5? : " + isDivisibleBy5(num));
//        2.
        int chickens, cows, pigs;
        System.out.print("Enter the number of chickens: ");
        chickens = scanner.nextInt();
        System.out.print("Enter the number of cows: ");
        cows = scanner.nextInt();
        System.out.print("Enter the number of pigs: ");
        pigs = scanner.nextInt();

        System.out.println("Total legs of " + chickens + " chickens, " + cows + " Cow and " + pigs + " pig : " + totalLegs(chickens, cows, pigs));
//        3.
        System.out.print("Enter a number: ");
        num = scanner.nextInt();
        System.out.println(num + " incremented by 1 : " + addOne(num));
    }

    //1.
    static Boolean isDivisibleBy5(int num) {
        return num % 5 == 0;
    }

    //2.
    static int totalLegs(int chickens, int cows, int pigs) {
        return chickens * 2 + cows * 4 + pigs * 4;
    }

    //3.
    static int addOne(int num) {
        return num + 1;
    }
}

File: src/com/riadsafowan/kotlin/Practice2.kt
package com.riadsafowan.kotlin

import java.util.*

fun main() {
//   1.
    var num: Int
    val scanner = Scanner(System.`in`)
    print("Enter a number: ")
    num = scanner.nextInt()
    println("Is the number " + num + " divisible by 5? : " + isDivisibleBy5(num))

    //  2.
    print("Enter the number of chickens: ")
    val chickens: Int = scanner.nextInt()
    print("Enter the number of cows: ")
    val cows: Int = scanner.nextInt()
    print("Enter the number of pigs: ")
    val pigs: Int = scanner.nextInt()
    println("Total legs of $chickens chickens, $cows Cow and $pigs pig : " + totalLegs(chickens, cows, pigs))

    //   3.
    print("Enter a number: ")
    num = scanner.nextInt()
    println(num.toString() + " incremented by 1 : " + addOne(num))
}

//1.
fun isDivisibleBy5(num: Int): Boolean {
    return num % 5 == 0
}

//2.
fun totalLegs(chickens: Int, cows: Int, pigs: Int): Int {
    return chickens * 2 + cows * 4 + pigs * 4
}

//3.
fun addOne(num: Int): Int {
    return num + 1
}
